apps/cli/src/commands/run.ts

```diff
@@ -71,17 +71,13 @@ export default class Run extends Command {
             ANVIL_VERBOSITY: flags.verbose ? "--steps-tracing" : "--silent",
             BLOCK_TIME: blockInterval.toString(),
             BLOCK_TIMEOUT: (blockInterval + 3).toString(),
-            CHAIN_ID: "31337",
-            EPOCH_DURATION: epochDuration.toString(),
-            MNEMONIC: DEFAULT_DEVNET_MNEMONIC,
```

Comment:
Remove unused DEFAULT_DEVNET_MNEMONIC variable from the code

Reply:
Compilation broke

Reply:
removed too much :sweat: 

push forced on last fixup